components/brave_wallet/browser/zcash/zcash_transaction_complete_manager.cc

```diff
@@ -54,9 +54,9 @@ void ZCashTransactionCompleteManager::CompleteTransaction(
     const ZCashTransaction& transaction,
     const mojom::AccountIdPtr& account_id,
     CompleteTransactionCallback callback) {
-  zcash_wallet_service_->zcash_rpc().GetLatestBlock(
+  zcash_wallet_service_->zcash_rpc().GetLightdInfo(
       chain_id,
-      base::BindOnce(&ZCashTransactionCompleteManager::OnGetLatestBlockHeight,
+      base::BindOnce(&ZCashTransactionCompleteManager::OnGetLightdInfo,
                      weak_ptr_factory_.GetWeakPtr(),
```

Comment:
How frequently branch id changes? Does it make sense to cache it per browser instance, or persist it per network?

Reply:
Several times a year i guess, we have to check whether it has been changed anyway